checkov/terraform/tf_parser.py

```diff
@@ -329,6 +330,67 @@ def parse_hcl_module(
 
         return module, tf_definitions
 
+    '''
```

Comment:
the doc should be under the function not above (so line 350 here after the definition).
Otherwise the documentation is good, but I will add that `terraform` also works per directory so that's the proper way to resolve it anyway